src/Exceptions/TerminusException.php

```diff
@@ -24,7 +27,7 @@ class TerminusException extends \Exception
    */
     public function __construct(
         $message = null,
-        $replacements = array(),
+        array $replacements = [],
```

Comment:
I'm curious why you're adding a type hint here when we're not doing that pretty much anywhere else? I'm not against using them, but we should be clear on what the accepted rule is for them.